language/move-prover/boogie-backend/src/bytecode_translator.rs

```diff
@@ -1260,6 +1262,26 @@ impl<'env> FunctionTranslator<'env> {
                             str_local(src)
                         );
                     }
+                    CastU16 => {
+                        let src = srcs[0];
+                        let dest = dests[0];
+                        emitln!(
+                            writer,
+                            "call {} := $CastU16({});",
+                            str_local(dest),
+                            str_local(src)
+                        );
+                    }
+                    CastU32 => {
+                        let src = srcs[0];
+                        let dest = dests[0];
+                        emitln!(
+                            writer,
+                            "call {} := $CastU32({});",
+                            str_local(dest),
+                            str_local(src)
+                        );
+                    }
```

Comment:
Could you also add the implementation of CastU256? Thanks!

Reply:
Done!